Make gene filtering optional in core workflow (#307)
AlexsLemonade/scpca-downstream-analyses: README.md

| `seed` | an integer to be used to set a seed for reproducibility when running the workflow | 2021 |
| `filtering_method` | `filtering_method`, the specified filtering method which can be one of "miQC" or "manual". For more information on choosing a filtering method, see [Filtering low quality cells](./additional-docs/processing-information.md#filtering-low-quality-cells) in the [processing information documentation](./additional-docs/processing-information.md) | "miQC" |
| `prob_compromised_cutoff` | the maximum probability of a cell being compromised as calculated by [miQC](https://bioconductor.org/packages/release/bioc/html/miQC.html), which is required when the `filtering_method` is set to `miQC` in the project metadata | 0.75 |
| `filter_genes` | a binary value indicating whether or not to perform gene fitering | `FALSE` |

> | `gene_detected_row_cutoff` | the percent of cells a gene must be detected in; genes detected are filtered regardless of the `filtering_method` specified in the project metadata | 5 |

Reviewer: ```suggestion
| `gene_detected_row_cutoff` | the percent of cells a gene must be detected in; genes detected are only filtered if `filter_genes` is set to `TRUE` | 5 |
```
We should modify this statement for this parameter and the `gene_means_cutoff` below.
